migrations/Version20231101141637.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20231101141637 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Removes Awards, unimplemented code.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('DROP SEQUENCE award_id_seq CASCADE');
+        $this->addSql('DROP SEQUENCE award_type_id_seq CASCADE');
+        $this->addSql('ALTER TABLE award DROP CONSTRAINT fk_8a5b2ee7a76ed395');
+        $this->addSql('ALTER TABLE award DROP CONSTRAINT fk_8a5b2ee73eb84a1d');
+        $this->addSql('ALTER TABLE award DROP CONSTRAINT fk_8a5b2ee7c54c8c93');
+        $this->addSql('DROP TABLE award_type');
+        $this->addSql('DROP TABLE award');
+        // Dropped Due to src/Entity/ApActivity.php, needs testing
+        // $this->addSql('ALTER TABLE ap_activity DROP CONSTRAINT fk_68292518a76ed395');
```

Comment:
Yea noticed that one. Unsure why it relies on the award for the foreign key though on this one. Thoughts on a fix for this one?

Reply:
There was a "many-on-one" DB relation as you can see in ApActivity.php between this `ap_activity` table and the **users** as well as the `ap_activity` and the **magazines**.

I think dropping `ALTER TABLE ap_activity DROP user_id` & `ALTER TABLE ap_activity DROP magazine_id` might be fine .. let me check the tables in DB later today..

Reply:
I would've expected the FK on `ap_activity` to `user_id` on `user` to help make sure all activitypub events correspond to a local user. I'm not really understanding what it has to do with awards :thinking:

Reply:
@e-five256 yea I didn't check the DB to be honest. but if the `user_id` and `magazine_id` has a relation only towards the awards table it can be removed. If the same foreign keys are used for other reasons, then not. 

Again looking at the change of @lilfade he only remove the Doctrine annotations, which is NOT just comments. So if you wish the delove this many to one relation related to the `awards` table  (`inversedBy`). You need to remove also the `public User $user;` and `public ?Magazine $magazine;` lines below the Doctrine annotations.

Meaning all these lines then need to be removed (but keep `subject_id`):
```
    #[ManyToOne(targetEntity: User::class, inversedBy: 'awards')]
    #[JoinColumn(nullable: false, onDelete: 'CASCADE')]
    public User $user;
    #[ManyToOne(targetEntity: Magazine::class, inversedBy: 'awards')]
    #[JoinColumn(nullable: true, onDelete: 'CASCADE')]
    public ?Magazine $magazine;
  ```
  
If you want to keep the `user_id` and `magazine_id` then you need to understand why this data was related to the `awards` table. See: https://www.doctrine-project.org/projects/doctrine-orm/en/2.16/reference/association-mapping.html#one-to-many-self-referencing

Reply:
I also checked the data in my database for this `ap_activity` table. I don't have any rows.. Which this table used at all??

```sql
SELECT * FROM ap_activity;
 id | user_id | magazine_id | subject_id | type | body | created_at 
----+---------+-------------+------------+------+------+------------
(0 rows)
```